pkg/network/config/config_test.go

```diff
@@ -157,6 +157,31 @@ func TestEnableHTTPMonitoringViaHTTP(t *testing.T) {
 	})
 }
 
+func TestEnableJavaTLSSupport(t *testing.T) {
```

Comment:
add a test to ensure javatls is disabled by default (it'll prevent us from accidentally enabling it by default)